src/api/forms/service.js

```diff
@@ -189,6 +189,56 @@ export async function updateDraftFormDefinition(formId, definition, author) {
   }
 }
 
+/**
+ * @param {string} formId - ID of the form
+ * @param {Partial<FormMetadataInput>} formUpdate - full JSON form definition
+ * @returns {Promise<string>}
+ */
+// TODO: author to be used to update updatedBy in next PR
+export async function updateFormMetadata(formId, formUpdate) {
+  logger.info(`Updating form metadata (draft) for form ID ${formId}`)
+
+  try {
+    // Get the form metadata from the db
+    const form = await getForm(formId)
+
+    if (form.live && formUpdate.title) {
+      throw Boom.badRequest(
+        `Field ${formUpdate.title} cannot be updated once the form has gone live`
+      )
+    }
+
+    const updatedSlug = slugify(formUpdate.title)
+
+    const updatedForm = {
+      ...formUpdate,
+      ...(formUpdate.title && { slug: updatedSlug })
+    }
+
+    const session = client.startSession()
+
+    try {
+      await session.withTransaction(async () => {
+        // Update the form metadata
+        await formMetadata.update(formId, { $set: updatedForm }, session)
+
+        logger.info(`Updating form metadata (draft) for form ID ${formId}`)
+      })
+
+      return updatedSlug
+    } finally {
+      await session.endSession()
+    }
+  } catch (err) {
+    logger.error(
+      err,
+      `Updating form metadata (draft) for form ID ${formId} failed`
```

Comment:
`(draft)` can be removed from here, since these changes do not apply to the draft and live.